bench-run/src/main/scala/dotty/tools/benchmarks/specialization/Functions.scala

```diff
@@ -0,0 +1,55 @@
+package dotty.tools.benchmarks.specialization
+
+import org.openjdk.jmh.annotations._
+import scala.util.Random
+
+class Functions {
+  extension (x: Int)
+    inline def times(op: => Unit): Unit = {
```

Comment:
```suggestion
    inline def times(inline op: Unit): Unit = {
```
to avoid an extra indirection to a `def` binding.

Reply:
Depends on #9753

Reply:
Do you compile these with fatal-warnings?

Reply:
If not you should be able to use it. It will generate the correct code but emit an incorrect warning.

Reply:
We enabled fatal warnings for all sub-projects.